Userland/Libraries/LibWeb/Painting/PaintableBox.h

```diff
@@ -123,6 +123,8 @@ class PaintableBox : public Paintable {
 
     bool is_out_of_view(PaintContext&) const;
 
+    void set_style_overrides(const CSS::ComputedValues& style_overrides) { m_style_overrides = style_overrides; };
```

Comment:
This makes each paint tree node significantly larger (each `ComputedValues` is already ~2300 bytes).
It seems unfortunate to add all that extra weight *just* to override some border properties.

Reply:
Makes sense. Separate structure just for the border properties, then?

Reply:
Made it a `BordersData` since it matches exactly what we need.